src/chrome/content/rules/Bitcoin.pl.xml

```diff
@@ -1,4 +1,9 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://bitcoin.pl/login => https://bitcoin.pl/login: None
```

Comment:
It's not detecting mixed content from FB. Mixed content checking is not yet supported. The 'None' is a bug in the checker when it attempts to calculate distance between two docs and one doc doesn't parse. I will fix.